cdap-examples/LogAnalysis/src/test/java/co/cask/cdap/examples/loganalysis/LogAnalysisAppTest.java

```diff
@@ -57,7 +50,7 @@
   private static final String TOTAL_HITS_VALUE = "2";
   private static final String TOTAL_RESPONSE_VALUE = "2";
   private static final String RESPONSE_CODE = "200";
-  private static final String TPFS_RESULT = "127.0.0.1:2";
+  private static final String TPFS_RESULT = "127.0.1.1:1\n127.0.0.1:2\n";
```

Comment:
how did the expected results change? I don't see a change input data or the program... How did the test pass before?